src/logger-coralogix.js

```diff
@@ -38,14 +55,15 @@ function createCoralogixLogger(config, params) {
     const [, , owPackage] = actionName.split('/');
     const applicationName = CORALOGIX_APPLICATION_NAME || namespace;
     const subsystemName = CORALOGIX_SUBSYSTEM_NAME || owPackage || 'n/a';
+    const sanitizedParams = sanitize(params);
```

Comment:
the params are not logged here at all...and the headers never were, due to a bug. so I would remove it completely.

Reply:
? I do not understand this comment. 2 lines below, I log the `__ow_headers` and this works for the dispatch action, which in my case (see https://github.com/adobe/helix-dispatch/blob/alex-playground/src/index.js#L77) "propagates" the headers to all the other action calls which also receive the headers and can log them.